src/plugins/hiddenColumns/test/publicAPI.e2e.js

```diff
@@ -210,5 +210,31 @@ describe('HiddenColumns', () => {
         expect(emptyData).toEqual(empyDataComparision);
       });
     });
+
+    describe('Data change by setDataAtCell()', () => {
```

Comment:
Just `setDataAtCell()` would be enough to describe what is inside (and would be consistent with the rest).

Reply:
Done by https://github.com/handsontable/handsontable/pull/7396/commits/2d125a63c7c2b05e866b202b6e6b7ec847720f85.